framework/DataObjects/PointSet.py

```diff
@@ -117,6 +117,22 @@ def _updateSpecializedInputValue(self,name,value,options=None):
       @ In,  value, float, newer value (single value)
       @ Out, None
     """
+
```

Comment:
Similarly, does this change have any affect on the rework branch, since this isn't the XPointSet?

Reply:
this is caused by the merging of devel on the top of this branch.